core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -252,11 +295,15 @@ oppia.controller('ExplorationHistory', [
   // Check if valid versions were selected
   $scope.areCompareVersionsSelected = function() {
     return (
-      $scope.compareVersions &&
-      $scope.compareVersions.hasOwnProperty('selectedVersion1') &&
-      $scope.compareVersions.hasOwnProperty('selectedVersion2'));
+      $scope.compareVersions && ($scope.selectedVersionsArray.length === 2));
   };
 
+  $scope.isCheckboxDisabled = function(versionNumber) {
+    if ($scope.selectedVersionsArray.length === 2) {
+      return !($scope.selectedVersionsArray.indexOf(versionNumber) > -1);
```

Comment:
I think it's clearer to write
    return $scope.selectedVersionsArray.indexOf(versionNumber) == -1;
here.